### PR TITLE
fix: AssetFailureReport propagation

### PR DESCRIPTION
**Pull request by SdgJlbl**
## Checklist

- [ ] [changelog](../CHANGELOG.md) was updated with notable changes
- [ ] documentation was updated

### CODE REVIEW

File: backend/api/events/sync.py [hunk 1]
```diff
@@ -380,27 +381,45 @@ def _disable_model(key: str) -> None:
...
-    logger.debug("Syncing failure report create", asset_key=event["asset_key"], event_id=event["id"])
+    logger.debug(
+        "Syncing failure report create",
+        asset_key=event["asset_key"],
+        event_id=event["id"],
+        failure_report=event["failure_report"],
+    )
 
     asset_key = event["asset_key"]
     failure_report = event["failure_report"]
```
[the remark this answers is not in the document]
Comment (SdgJlbl) [reply backend/api/events/sync.py:394]: I'd rather keep the initial order, actually, to match all other sync functions, if that's okay with you.

[the remark this answers is not in the document]
Comment (thbcmlowk) [reply backend/api/events/sync.py:394]: Of course it is

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Comment by Owlfred** [comment]
End to end tests: :hourglass_flowing_sand: **STARTING**

**Pull request merged**